packages/eventemittable/__tests__/eventemittable.js

```diff
@@ -0,0 +1,197 @@
+        ee.on('foo', listener);
+        ee.emit('foo');
+        ee.emit('foo');
+        expect(listener.mock.calls.length)
+          .toBe(2);
+      });
+    });
+
+    describe('emit()', function () {
+      var listeners;
+
+      beforeEach(function () {
+        listeners = [
+          jest.fn(),
+          jest.fn(),
+          jest.fn()
+        ];
+        listeners.forEach(function (listener) {
+          ee.once('foo', listener);
+        });
+      });
+
+      it('should emit to multiple listeners', function () {
+        ee.emit('foo');
+
+        listeners.forEach(function (listener) {
+          expect(listener.mock.calls.length)
+            .toBe(1);
+        });
+      });
+    });
+
+    describe('removeListener()', function () {
+      var listeners;
+
+      beforeEach(function () {
+        listeners = [
+          jest.fn(),
+          jest.fn()
+        ];
+        listeners.forEach(function (listener) {
+          ee.once('foo', listener);
+        });
+      });
+
+      it('should remove a listener from the event', function () {
+        ee.removeListener('foo', listeners[1]);
+        ee.emit('foo');
+        expect(listeners[0].mock.calls.length)
+          .toBe(1);
+        expect(listeners[1].mock.calls.length)
+          .toBe(0);
+      });
+    });
+
+    describe('removeAllListeners()', function () {
+      var listeners;
+
+      beforeEach(function () {
+        listeners = [
+          jest.fn(),
+          jest.fn()
+        ];
+        listeners.forEach(function (listener) {
+          ee.once('foo', listener);
+          ee.once('bar', listener);
+        });
+      });
+
+      describe('when provided an event', function () {
+        it('should remove all listeners from a single event', function () {
+          ee.removeAllListeners('foo');
+          ee.emit('foo');
+          listeners.forEach(function (listener) {
+            expect(listener.mock.calls.length)
+              .toBe(0);
+          });
+        });
+      });
+
+      describe('when not provided an event', function () {
+        it('should remove all listeners from all events', function () {
+          ee.removeAllListeners();
+          ee.emit('foo');
+          ee.emit('bar');
+          listeners.forEach(function (listener) {
+            expect(listener.mock.calls.length)
+              .toBe(0);
+          });
+        });
+      });
+    });
+
+    describe('getMaxListeners()', function () {
+      it('should return the (default) maximum number of listeners',
+        function () {
+          expect(ee.getMaxListeners())
+            .toBe(EventEmitter.defaultMaxListeners);
+        });
+    });
+
+    describe('setMaxListeners()', function () {
+      it('should set the maximum number of listeners', function () {
+        ee.setMaxListeners(2);
+        expect(ee.getMaxListeners())
+          .toBe(2);
+      });
+    });
+
+    describe('listeners()', function () {
+      var listeners;
+
+      beforeEach(function () {
```

Comment:
Why to actually use `beforeEach` for a single test? Feels like some weird practice :) Same for others above.

Generally it's much easier and predictable to create a function that does setup and call it from the test. Instead of relying on closure values and possible conflicts between tests, you can just return values from those helper functions which ensures that every test has it's own piece.